components/inputs/demo/input-number.html

```diff
@@ -56,10 +56,10 @@ <h2>Min and Max Values</h2>
 				</template>
 			</d2l-demo-snippet>
 
-			<h2>Step</h2>
+			<h2>Min and Max Fraction Digits</h2>
 			<d2l-demo-snippet>
 				<template>
-					<d2l-input-number label="Grade" style="width: 5rem;" value="85.76" step="0.01"></d2l-input-number>
+					<d2l-input-number label="Points" value="12345.67" min-fraction-digits="1" max-fraction-digits="2" style="width: 6rem;"></d2l-input-number>
```

Comment:
nit: It'd be nice to remove the `style="width: 6rem"` from the demo. The reason being that people often copy and paste from demo examples.